README.md

```diff
@@ -40,6 +40,7 @@ By adding your checkbox, you are approving [DevOps Malmö Meetup group guideline
 - [x] *2018-04-09* - [Learn Ansible automation with Red Hat](https://goo.gl/ThoJBB) by Magnus Glatz
 - [x] *2018-02-20* - [DevOps Skåne - introduction for students](https://goo.gl/K5sbk7) by Karen Har-yan, Fredrik Stål and Albert Rigo
 - [x] *2018-01-15* - [Basic Kubernetes](https://goo.gl/GTHwyi) by Karen Har-yan
+- [x] *2021-03-09* - Automation is Hard and we are doing it wrong
```

Comment:
Please add this line to the top ;)

Reply:
I mangled my history to make this change.